pepper-apis/src/main/java/org/broadinstitute/ddp/route/GetDsmMedicalRecordRoute.java

```diff
@@ -96,7 +100,7 @@ public Participant handle(Request request, Response response) {
 
             // Converting booleans to ints because it's the format DSM expects
             MedicalRecordService.ParticipantConsents consents = medicalRecordService.fetchBloodAndTissueConsents(
-                    handle, dsmParticipant.getUserId(), dsmParticipant.getUserGuid(), studyDto.getId(), studyGuid);
+                    handle, dsmParticipant.getUserId(), dsmParticipant.getUserGuid(), operatorGuid, studyDto.getId(), studyGuid);
```

Comment:
`GetDsmMedicalRecordRoute` is called by DSM, one of the other subsystem of the platform, so there's no operator here.

Reply:
Done